rosbag2_cpp/test/rosbag2_cpp/test_sequential_writer.cpp

```diff
@@ -314,8 +319,13 @@ TEST_F(SequentialWriterTest, do_not_use_cache_if_cache_size_is_zero) {
 
   std::string rmw_format = "rmw_format";
 
+  std::string msg_content = "Hello";
```

Comment:
This test ensures that cache is not used if cache size is 0 and previously did not care about the `serialized_data` in the message. However, I thought it is better if we do test with a close to real world scenario where the message has some data as well.